retirement/serializers.py

```diff
@@ -428,7 +422,15 @@ def validate(self, attrs):
         active_reservations = Reservation.objects.filter(
             user=validated_data['user'],
             is_active=True,
-        ).exclude(**validated_data).values_list(
+        )
+
+        # deletes reservations with the same user and retirement from
+        # active_reservation
+        # keeping this line allows to reserves two time the same retirement
+        #
+        # active_reservations = active_reservations.exclude(**validated_data)
```

Comment:
Why do we keep this comment here ?

Reply:
Because I am not that confident that we won't need it in the future..... But it seems like the right decision for now